### ISSUE TITLE
Multilabel Bert: Architecture Question

### ISSUE THREAD

**Issue by sb1992** [ThilinaRajapakse/simpletransformers#835]
As I understand the difference between Multiclass BERT and Mulitlabel BERT is that we use BCElogitloss() instead of cross entropy as our loss function and to get the labels we use sigmoid? My question is shouldn't we be also using a different MLP (classification) for each class in a multi-label setting? Like if we are not using it could you please explain why we don't need to use a separate MLP for each class (with BCE loss()) after getting the CLS token?
